server/bleep/src/semantic.rs

```diff
@@ -193,6 +197,13 @@ impl Semantic {
                 .build()?,
         );
 
+        #[cfg(windows)]
+        {
+            unsafe {
+                ort::ort().DisableTelemetryEvents.as_ref().unwrap()(environment.env_ptr());
+            }
+        }
+
```

Comment:
I'd really like this unsafe code to move to our `ort` fork, so we can upstream it once we're happy. It is a single function, so not a huge change on either side.

Reply:
Swapped this with a new branch that adds `EnvBuilder::with_telemetry`